src/ui/src/components/core/content/CoreDataframe/CoreDataframeCellNumber.vue

```diff
@@ -20,14 +20,16 @@ function onChange(newValue: number) {
 </script>
 
 <template>
-	<WdsNumberInput
-		v-if="editable"
-		class="CoreDataframeCellNumber--input"
-		:model-value="Number(value)"
-		@focusout="onChange($event)"
-	/>
-	<div v-else class="CoreDataframeCellNumber--text">
-		{{ value }}
+	<div>
+		<WdsNumberInput
+			v-if="editable"
+			class="CoreDataframeCellNumber--input"
+			:model-value="Number(value)"
+			@focusout="onChange($event)"
+		/>
+		<div v-else class="CoreDataframeCellNumber--text">
+			{{ value }}
+		</div>
 	</div>
```

Comment:
I'm not sure that you need the extra `div` in read-only mode

```suggestion
	<div v-if="editable">
		<WdsNumberInput
			class="CoreDataframeCellNumber--input"
			:model-value="Number(value)"
			@focusout="onChange($event)"
		/>
	</div>
	<div v-else class="CoreDataframeCellNumber--text">
		{{ value }}
	</div>
```